CHANGELOG.md

```diff
@@ -4,6 +4,13 @@
 
 [Unreleased]: https://github.com/chaostoolkit/chaostoolkit-kubernetes/compare/0.26.2...HEAD
 
+## [0.26.3][] - 2022-04-05
```

Comment:
I should have mentionned this. When you update the changelog, don't set a new version. Pushing a version occurs at a later time. Please remove this line for now.